src/registrar/models/domain.py

```diff
@@ -7,7 +7,8 @@
 from django_fsm import FSMField, transition  # type: ignore
 
 from api.views import in_domains
-from epp.mock_epp import domain_info, domain_check
+
+# from epplibwrapper import CLIENT as registry, commands
```

Comment:
**question (non-blocking)**: We don't usually keep commented-out code in our files. If it makes sense here, could we also include an explanatory comment above about why it appears here?

Reply:
Was merely an example of how the wrapper will be imported. I've removed it.